src/amo/actions/reviews.js

```diff
@@ -169,7 +169,7 @@ type FetchReviewsParams = {|
   addonSlug: string,
   errorHandlerId: string,
   page?: string,
-  score?: string,
+  score: string | null,
```

Comment:
can this be optional? for example on the reviews landing page, right? we wouldn't need this..